src/lib/bootloader/generic_widgets.rb

```diff
@@ -73,20 +72,6 @@ def handle
         return :redraw if !Yast::Popup.ContinueCancel(popup_msg)
       end
 
-      if new_bl == "systemd-boot"
-        # popup - Continue/Cancel
-        popup_msg = _(
-          "\n" \
-          "Systemd-boot support is currently work in progress and\n" \
```

Comment:
We have implemented systemd-boot last year and have not gotten strong errors.
As our team (Ähm, mainly I am :-) ) is maintainer of the systemd-boot part in yast-bootloader we would suggest to remove this warning.